eppoclient/rules.go

```diff
@@ -140,13 +152,21 @@ func isOne(attributeValue interface{}, s string) bool {
 		if err != nil {
 			return false
 		}
-		return promoteInt(attributeValue) == value
+		promotedValue, err := promoteInt(attributeValue)
+		if err != nil {
+			return false
+		}
+		return promotedValue == value
```

Comment:
minor: `promotedValue == value` suggest that they are the same value. I would rather rename `promotedValue` to `promotedAttributeValue` or similar.